__tests__/src/components/TicketsTable.test.tsx

```diff
@@ -97,4 +97,92 @@ describe('TicketsTable', () => {
     render(<TicketsTable items={[]} />);
     expect(screen.getByText('Urgent')).toBeInTheDocument();
   });
+
+  it('renders Urgent on table if a case is Urgent', () => {
+    render(<TicketsTable items={[]} />);
+    expect(screen.getByText('Urgent')).toBeInTheDocument();
+  });
+
+  it('should sort the table when column headers are clicked', () => {
+    const mockItemsSort: ServiceRequestSummary[] = [
+      {
+        id: '1',
+        pet: 'Dog',
+        urgent: true,
+        description: 'Annual checkup',
+        created_at: '2023-05-01T12:00:00Z',
+        team_member: { first_name: 'John', email: 'john@example.com' },
+        client: 'Client 1',
+        status: 'Open',
+        modified_at: '2023-05-01T12:00:00Z',
+      },
+      {
+        id: '2',
+        pet: 'Cat',
+        urgent: false,
+        description: 'Rabies shot',
+        created_at: '2023-05-02T12:00:00Z',
+        team_member: { first_name: 'Jane', email: 'jane@example.com' },
+        client: 'Client 2',
+        status: 'Open',
+        modified_at: '2023-05-02T12:00:00Z',
+      },
+      {
+        id: '3',
+        pet: 'Bird',
+        urgent: false,
+        description: 'Wing repair',
+        created_at: '2023-05-03T12:00:00Z',
+        team_member: { first_name: 'Bob', email: 'bob@example.com' },
+        client: 'Client 3',
+        status: 'Open',
+        modified_at: '2023-05-03T12:00:00Z',
+      },
+    ];
+
+    render(<TicketsTable items={mockItemsSort} />);
+
+    // Test sorting by Owner (pet field)
+    // const ownerHeader = screen.getByText('Owner');
```

Comment:
Nit: can you remove this commented out code?

Reply:
Or should this be uncommented?

Reply:
Opps was testing values should be commented in